README.md

```diff
@@ -47,38 +37,31 @@ As a customer
 So that I am reassured that my order will be delivered on time
 I would like to receive a text such as "Thank you! Your order was placed and will be delivered before 18:52" after I have ordered
 ```
+**My Approach**
+-----
+
+**Domain Model Diagrams**
+
+
+**Responsibilities**
+
+![Responsibilities](https://github.com/Sumner1185/takeaway-challenge/blob/master/ResponsibilityTable.png)
```

Comment:
Really nice format for the class diagram for the responsibilities of each class! Is the Takeaway class responsible for allowing the order to be placed or sending the text message? Usually responsibilities can be derived by looking at the public methods within a class